graphql_compiler/compiler/filters.py

```diff
@@ -650,4 +662,5 @@ def process_filter_directive(filter_operation_info, location, context):
         FilterInfo(fields=fields, op_name=op_name, args=tuple(operator_params))
     )
 
+    context['directive_location'] = location
```

Comment:
Hacky. I'm not sure if this is the intended use of the context argument.

Reply:
Why not just weave this into the filter operator functions, and then into the `_represent_argument`? This seems really hacky, and unnecessarily so.